code/modules/client/preferences.dm

```diff
@@ -1309,6 +1311,9 @@ GLOBAL_LIST_INIT(bgstate_options, list(
 				if("pred_mask_type")
 					var/new_predator_mask_type = tgui_input_number(user, "Choose your mask type:\n(1-19)", "Mask Selection", 1, 19, 1)
 					if(new_predator_mask_type) predator_mask_type = floor(text2num(new_predator_mask_type))
+				if("pred_accessory_type")
+					var/new_predator_accessory_type = tgui_input_number(user, "Choose your mask accessory type:\n(0-1)", "accessory Selection", 0, 1, 0)
+					if(new_predator_accessory_type) predator_accessory_type = floor(text2num(new_predator_accessory_type))
```

Comment:
```suggestion
					if(new_predator_accessory_type)
						predator_accessory_type = floor(text2num(new_predator_accessory_type))
```
also validate this is between 0-1. and it should be sanitized. and it should be saved in prefs